templates/quickstart-bitbucket-dc-with-vpc.template.yaml

```diff
@@ -382,6 +385,14 @@ Parameters:
       - 'PostgreSQL'
     ConstraintDescription: Must be 'Amazon Aurora PostgreSQL' or 'PostgreSQL'.
     Type: String
+  DBEngineVersion:
+    Default: 11
+    AllowedValues:
+      - 10
+      - 11
+      - 12
+    Description: "The database engine version to use; we'll install a suitable minor version for your chosen engine. Make sure that the Jira version you're installing supports the database engine selected."
```

Comment:
@ddomingorh, we will need to update this and link it to Bitbucket support documentation. I am of the opinion that we should document that Aurora doesn't support v 12 as CloudFormation doesn't have the ability to prevent the parameter validation based on another parameter value.